shotover-proxy/tests/helpers/cassandra.rs

```diff
@@ -298,18 +609,61 @@ impl ResultValue {
+    pub fn new_from_cdrs(value: CassandraType, version: Version) -> Self {
+        match value {
+            CassandraType::Ascii(ascii) => ResultValue::Ascii(ascii),
+            CassandraType::Bigint(big_int) => ResultValue::BigInt(big_int),
+            CassandraType::Blob(blob) => ResultValue::Blob(blob.into_vec()),
+            CassandraType::Boolean(b) => ResultValue::Boolean(b),
+            CassandraType::Counter(counter) => ResultValue::Counter(counter),
+            CassandraType::Decimal(decimal) => {
+                ResultValue::Decimal(decimal.serialize_to_vec(version))
+            }
+            CassandraType::Double(double) => ResultValue::Double(double.into()),
+            CassandraType::Float(float) => ResultValue::Float(float.into()),
+            CassandraType::Int(int) => ResultValue::Int(int),
+            CassandraType::Timestamp(timestamp) => ResultValue::Timestamp(timestamp),
+            CassandraType::Uuid(uuid) => ResultValue::Uuid(uuid),
+            CassandraType::Varchar(varchar) => ResultValue::Varchar(varchar),
+            CassandraType::Varint(var_int) => ResultValue::VarInt(var_int.to_signed_bytes_be()),
+            CassandraType::Timeuuid(uuid) => ResultValue::TimeUuid(uuid),
+            CassandraType::Inet(ip_addr) => ResultValue::Inet(ip_addr.to_string()),
+            CassandraType::Date(date) => ResultValue::Date(date.serialize_to_vec(version)),
+            CassandraType::Time(time) => ResultValue::Time(time.serialize_to_vec(version)),
+            CassandraType::Smallint(small_int) => ResultValue::SmallInt(small_int),
+            CassandraType::Tinyint(tiny_int) => ResultValue::TinyInt(tiny_int),
+            CassandraType::Duration(duration) => {
+                ResultValue::Duration(duration.serialize_to_vec(version))
+            }
+            CassandraType::List(list) => {
+                let mut elements = Vec::new();
+                for element in list {
+                    elements.push(ResultValue::new_from_cdrs(element, version));
+                }
+                ResultValue::List(elements)
+            }
+            CassandraType::Map(map) => {
+                let mut elements = Vec::new();
+                for (k, v) in map {
+                    elements.push((
+                        ResultValue::new_from_cdrs(k, version),
+                        ResultValue::new_from_cdrs(v, version),
+                    ));
+                }
+                ResultValue::Map(elements)
+            }
+            CassandraType::Set(set) => {
+                let mut elements = Vec::new();
+                for element in set {
+                    elements.push(ResultValue::new_from_cdrs(element, version));
+                }
+                ResultValue::Set(elements)
+            }
+            CassandraType::Udt(_) => todo!(),
+            CassandraType::Tuple(_) => todo!(),
+            CassandraType::Null => todo!(),
```

Comment:
TODO's left

Reply:
Those were already there in the cpp impl. Could we not block this PR on those?

Reply:
null and tuple now have implementations for cpp and are very easy to implement here.
I very much agree with Udt though, dont implement that here.

Reply:
Sure, I'll implement null and tuple then.

Reply:
Where's the impl of tuple and null for cpp?

Reply:
![image](https://user-images.githubusercontent.com/5120858/190064151-02057280-fcf6-4adc-a065-a6ea264bcf8d.png)

Reply:
wow I cant believe rust didnt warn us about the duplicated ValueType::TUPLE :no_mouth: 
![image](https://user-images.githubusercontent.com/5120858/190064304-da25b7de-8865-41e9-a7e9-398da68493ab.png)

Reply:
Oh lol. Somehow both were hidden